sys_tests/__init__.py

```diff
@@ -0,0 +1,9 @@
+from cosmo_tester.framework.testenv import bootstrap, teardown
```

Comment:
do something like this
https://github.com/cloudify-cosmo/cloudify-openstack-plugin/blob/master/system_tests/__init__.py